provisioning/device/src/ShouldRetry.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
```

Comment:
Missing copyright and license information

Reply:
Good catch, will add